helpers.go

```diff
@@ -0,0 +1,117 @@
+// Copyright 2015 - António Meireles  <antonio.meireles@reformi.st>
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package main
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+)
+
+// returns true if payload not nil/empty
+func got(in interface{}) bool {
```

Comment:
I'm not really a huge fan of this function. It isn't all that idiomatic, it's usually just best to check what you want specifically at the caller location.